doc/manual/src/language/builtin-constants.md

```diff
@@ -14,9 +14,8 @@ These constants are built into the Nix language evaluator:
 
 - [`builtins.currentSystem`]{#builtins-currentSystem} (string)
 
-  The built-in value `currentSystem` evaluates to the Nix platform
-  identifier for the Nix installation on which the expression is being
-  evaluated, such as `"i686-linux"` or `"x86_64-darwin"`.
+  Return the system type of the Nix installation on which the expression is being evaluated.
+  This value is determined by the [`system` configuration setting](@docroot@/command-ref/conf-file.md#conf-system).
```

Comment:
Problems:
 - `system` doesn't _have_ to be the system of the Nix installation.
 - values don't return anything.

```suggestion
  The system type of the Nix installation on which the expression is being evaluated, or the [`system` configuration setting](@docroot@/command-ref/conf-file.md#conf-system) if set to a different value.
```

Did I keep this in the order of relevance or did I make it contrived?
At least it's more truthful.

Reply:
```suggestion
  The value of the [`system` configuration setting](@docroot@/command-ref/conf-file.md#conf-system).
```